docs/operations/data_migration.md

```diff
@@ -76,3 +76,55 @@ An example script using this technique is in
 ```shell
 docker compose run app ./manage.py load_domain_invitations /app/escrow_domain_contacts.daily.dotgov.GOV.txt /app/escrow_contacts.daily.dotgov.GOV.txt
 ```
+
+## Transition Domains
+Verisign provides information about Transition Domains in 3 files;
+FILE 1: escrow_domain_contacts.daily.gov.GOV.txt
+FILE 2: escrow_contacts.daily.gov.GOV.txt
+FILE 3: escrow_domain_statuses.daily.gov.GOV.txt
+
+Transferring this data from these files into our domain tables happens in two steps;
+
+### STEP 1: Load Transition Domain data into TransitionDomain table
+
+**SETUP**
+
+In order to use the management command, we need to add the files to a folder under `src/`.
+This will allow Docker to mount the files to a container (under `/app`) for our use.  
+
+ - Create a folder called `tmp` underneath `src/`
+ - Add the above files to this folder
+ - Open a terminal and navigate to `src/`
+
+Then run the following command  (This will parse the three files in your `tmp` folder and load the information into the TransitionDomain table);
+```shell
+docker compose run -T app ./manage.py load_transition_domain /app/tmp/escrow_domain_contacts.daily.gov.GOV.txt /app/tmp/escrow_contacts.daily.gov.GOV.txt /app/tmp/escrow_domain_statuses.daily.gov.GOV.txt
+```
+
+**OPTIONAL COMMAND LINE ARGUMENTS**: 
+`--debug`
+This will print out additional, detailed logs.
+
+`--limitParse 100` 
+Directs the script to load only the first 100 entries into the table.  You can adjust this number as needed for testing purposes.  
```

Comment:
Which file is used for limit parsing. really this only limits the file parsing for the last file, the one that has domain statuses right? This should be included here if I am correct as helps for manually testing.